test/OmenThumbnailMapping.t.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.13;
+
+import {Test, console} from "forge-std/Test.sol";
+import {OmenThumbnailMapping, IERC20} from "../src/OmenThumbnailMapping.sol";
+
+contract FixedProductMarketMaker is IERC20 {
+    // Mocked product maker that allows us to easily change balances (liquidity provided by the user).
+    mapping(address => uint256) private balances;
+
+    function setBalance(address account, uint256 balance) public {
+        balances[account] = balance;
+    }
+
+    function balanceOf(address account) external view override returns (uint256) {
+        return balances[account];
+    }
+}
+
```

Comment:
You could inherit from this (https://github.com/RevelationOfTuring/foundry-openzeppelin-contracts/blob/master/test/token/ERC20/utils/TokenTimelock/MockERC20.sol) instead, so you don't have to rewrite any methods.

Reply:
Seems like the same thing in the end, is there preference in web3 crowd to inherit stuff like this? 

In Python, I wouldn't install a new library to save a few lines of code.

Reply:
Actually forge dependencies are git submodules, see for ex this [starter repo](https://github.com/FrankieIsLost/forge-template/tree/master/lib).

My suggestion would be to install the open-zeppelin as dependency (the repo is [this](https://github.com/OpenZeppelin/openzeppelin-contracts)) (so that you don't need the ERC20 interface hardcoded), and then you can directly use [ERC20Mock.sol](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/mocks/token/ERC20Mock.sol) in the tests.

Reply:
I see how it works, but that doesn't answer my question unfortunatelly.